### PR TITLE
Add travis badge, update README

### PR DESCRIPTION
**Pull request by deric**
I've updated compiling instruction for fast skim readers :) Geeks cloning `htop` directly from github, can't use the `./configure` script. Is it ok this way?

P.S. would you mind moving the source code into subfolders, so that the repo would be in more github-like format? Anyway, it's a non issue, and thanks a lot for the awesome htop!

### CODE REVIEW

**Comment by Explorer09** [comment]
I don't believe configuring htop with `--prefix=/usr` on a production system is a good idea. `/usr` is supposed to contain distro packages and be maintained by distro's package managers. You will overwrite something if you are not careful. A better way is to tell the user to remove the distro package first, and then build and install from source.

**Comment by deric** [comment]
Right, I agree. It might depend on Linux distribution but, `/usr/local/bin` precedes `/usr/bin` on `$PATH`. So removing the distribution package might not be necessary.

**Comment by hishamhm** [comment]
Nice, thanks! 

But I wouldn't put instructions for a single distro here in the README (I don't use Debian/Ubuntu and don't wish to maintain those instructions myself, and there are many other downstream distros out there, I don't feel like promoting Debian/Ubuntu in particular, even though they do great work).

Also, ./autogen.sh is necessary when building from Git, but not when building from the tarball, and this README is included in both.

Apart from that, the other changes look good, thank you!

**Comment by deric** [comment]
@hishamhm github should show the `README.md` file by default. If we remove symlink to `README`, you can copy to tar file just the README with slightly different instructions. Is that ok?

**Comment by deric** [comment]
I can write build requirements for a few distributions, just give me a list of those you'd like to be mentioned. It might support the impression, that htop is a actually a multi-platform program :smiley:.

File: README [hunk 1]
```diff
@@ -1,50 +1,56 @@
...
+
+By default `make install` will install into `/usr/local`, for changing
+the path use `./configure --prefix=/some/path`.
+
+See the manual page (`man htop`) or the on-line help ('F1' or 'h'
+inside `htop`) for a list of supported key commands.
 
-When fetching the code from the development repository, you need
-to run the `./autogen.sh` script, which in turn requires autotools
-to be installed.
```
Comment (hishamhm) [inline README]: I appreciate having an easily copy-pastable command there, but this sentence is a bit more detailed then than the new version.

**Comment by hishamhm** [comment]
Merging! I'll tweak it further later if necessary :)

**Pull request merged**